packages/jest-cli/src/cli/args.js

```diff
@@ -34,6 +34,15 @@ const check = (argv: Object) => {
     );
   }
 
+  if (argv.findRelatedTests && argv._.length === 0) {
+    throw new Error(
+      '--findRelatedTests requires paths to be specified.\n' +
+      'Example usage: jest --findRealtedTests ./src/source.js',
+    );
```

Comment:
Let's do:

"Jest: The `findRelatedTests` option requires file paths to be specified. Example usage: `jest --findRelatedTests ./src/source.js ./src/index.js`."